src/js/component.js

```diff
@@ -38,11 +38,11 @@ import window from 'global/window';
  */
 class Component {
 
-  constructor(player, options, ready){
+  constructor(player, options, ready) {
 
     // The component might be the player itself and we can't pass `this` to super
     if (!player && this.play) {
-      this.player_ = player = this;
+      this.player_ = player = this; // eslint-disable-line
```

Comment:
This is how you would ignore a single line if we really want to keep it and it breaks the rules.
I decided to do it for this line because it's too much of a pain to figure out a good way to refactor this so what is happening is still clear.